backend/routes/tags/tags-route.js

```diff
@@ -0,0 +1,4 @@
+const express = require("express")
+const router = express.Router()
+
+// Add your code here
```

Comment:
I'm actually not sure it makes sense to have a separate router file specifically for tags, since tags will end up being part of the URI for other routers.

For example, here are examples of what the URI's might look like.
<img width="702" alt="Screen Shot 2022-09-02 at 3 29 35 PM" src="https://user-images.githubusercontent.com/36748732/188242748-0fada60d-bfea-464e-9fa4-5963e241d9fc.png">

Reply:
I think there should be separate tag routes because admin will view, add, or delete tags individually. Other routers contain only one copy of a tag just like label on github

Reply:
Ohhhh like would there be a database with the list of tags in use?

Reply:
yeah